app/src/main/java/nodomain/freeyourgadget/gadgetbridge/devices/qhybrid/TaskerPlugin.java

```diff
@@ -0,0 +1,1001 @@
+package nodomain.freeyourgadget.gadgetbridge.devices.qhybrid;
```

Comment:
This class looks like it is not really qhybrid specific, or is it? Where did you get it from? Who is the author and what's the license?

Reply:
It is mentioned to download in [this](https://tasker.joaoapps.com/plugins-intro.html) tutorial, but i will remove anything tasker-related anyway in favor of intents

Reply:
Alright.